README.md

```diff
@@ -140,7 +140,16 @@ If you are using JRuby then you need to download latest [Oracle JDBC driver](htt
 TESTS
 -----
 
-* Create Oracle database schema for test purposes. Review `spec/spec_helper.rb` to see default schema/user names and database names (use environment variables to override defaults)
+Review `spec/spec_helper.rb` to see default schema/user names and database names (use environment variables to override defaults)
+
+##### Prepare database
+
+* With local Vagrant based Oracle XE database
+
+        # Download Oracle XE database (oracle-xe-11.2.0-1.0.x86_64.rpm.zip) from Oracle Home page
+        vagrant up
+
+* Within other Oracle Database create Oracle database schema for test purposes.
 
         SQL> CREATE USER hr IDENTIFIED BY hr;
```

Comment:
Would be good to add a default tablespace:
SQL> CREATE USER hr IDENTIFIED BY hr DEFAULT TABLESPACE USERS;